internal/processor/filterset/factory/config.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2020 OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package factory
+
+const (
+	// REGEXP is the FilterType for filtering by regexp string matches.
+	REGEXP MatchType = "regexp"
+	// STRICT is the FilterType for filtering by exact string matches.
+	STRICT MatchType = "strict"
```

Comment:
We use MixedCaps for naming and avoid ALLCAPS in our codebase. This is also in line with what Go recommends: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps
Use MixedCaps.